composer.json

```diff
@@ -57,6 +56,11 @@
     "drupal/trusted_redirect": "^1.4",
     "drupal/token": "1.6",
     "drupal/vuejs": "^1.0@beta",
-    "drupal/simple_oauth": "^3.16"
+    "drupal/simple_oauth": "^3.16",
+    "thunder/thunder-distribution": "~3.3",
+    "drupal/media_entity": "^2.0",
```

Comment:
should not be added here, it's part of contentpool-data-model here.

Reply:
Yes, I also think so but contentpool-data-model  is not part of this composer project

Reply:
oh, I see - probably because it should be covered with thunder. nevermind, it does not hurt either here.